### PR TITLE
feat: VC Store Get Credential API along with Command/Controller Support

### LINKED ISSUE #1303: Controller APIs for VC Store
Build REST/JS Controller APIs:
- VC Store 
  - Save with friendly name
  - Get based on vcID
  - Get all VC names

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #1303 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: cmd/aries-js-worker/src/agent-rest-client.js [hunk 1]
```diff
@@ -107,6 +107,15 @@ const pkgs = {
             method: "GET",
             pathParam:"id"
         },
+        GetCredentialByName: {
+            path: "/verifiable/credential/name/{name}",
```
Comment (sudeshrshetty) [inline cmd/aries-js-worker/src/agent-rest-client.js]: URL is `/verifiable/credential/name/{name}??` 

why can't be `/verifiable/credential/{name}?` like connection by ID in didexchange controller?

Comment (rolsonquadras) [reply cmd/aries-js-worker/src/agent-rest-client.js]: we already have one API for ID - `/verifiable/credential/{id}`

Comment (sudeshrshetty) [reply cmd/aries-js-worker/src/agent-rest-client.js]: QueryConnectionByID: {
            path: "/connections/{id}",
            method: "GET",
            pathParam:"id"
        },
        QueryConnections: {
            path: "/connections",
            method: "GET"
        },

Comment (rolsonquadras) [reply cmd/aries-js-worker/src/agent-rest-client.js]: https://github.com/hyperledger/aries-framework-go/blob/master/cmd/aries-js-worker/src/agent-rest-client.js#L105-L118.

Comment (sudeshrshetty) [reply cmd/aries-js-worker/src/agent-rest-client.js]: I can see ` path: "/verifiable/credential/{id}"` taken of ID?

May be we can follow `QueryConnectionByID vs QueryConnections` pattern from didexchange (may be a followup?)

Comment (rolsonquadras) [reply cmd/aries-js-worker/src/agent-rest-client.js]: I don't think its applicable her. Here we have 3 APIs - GetByID, GetByName and GetAll.

**Pull request merged**